.github/actions/windows_install/action.yml

```diff
@@ -21,30 +21,62 @@ runs:
         echo $EXE
         $BIN_DIR=(Split-Path -Parent ${EXE}) # Find folder containing c compiler
         $MINGW_DIR=(Split-Path -Parent ${BIN_DIR}) # Find the enclosing mingw directory
-        $LIBRARY_DIR="$MINGW_DIR/lib"
-        echo $LIBRARY_DIR | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-        echo "LIBRARY_DIR=${LIBRARY_DIR}" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        echo $MINGW_DIR | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+        echo "MINGW_DIR=${MINGW_DIR}" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
 
```

Comment:
Now you are adding the enclosing `mingw` directory to GITHUB_PATH instead of just its `lib` subdirectory. Does this change make a difference, or is it just for better clarity in this action file?

Reply:
Definitely, it is because we installed the dynamic libraries into the lib/ folder

Reply:
I have removed this line: [6425fa2](https://github.com/pyccel/pyccel/pull/2294/commits/6425fa2d298e7fc8e1efdec1b800d9d7110f1312)